tensorflow/python/ops/array_ops.py

```diff
@@ -587,6 +587,9 @@ def _slice_helper(tensor, slice_spec, var=None):
   (isinstance(slice_spec, ops.Tensor) and slice_spec.dtype == dtypes.bool) or \
   (isinstance(slice_spec, np.ndarray) and slice_spec.dtype == bool):
     return boolean_mask(tensor=tensor, mask=slice_spec)
+  
+  if any(isinstance(i, list) for i in slice_spec):
```

Comment:
Do all the existing slice tests pass with this change?

Can you write new unit tests that specify the behavior you'd like to support? Ideally they would cover things such as a python list as an argument, a tensor as an argument, etc.

Reply:
@alextp, Thanks for your review. I haven't pass all the python test yet. I want to write some test to verify it, but I think I have to compile all the code and test it from the python API, would you please suggest some convenient way to tackle this?